Building.md

```diff
@@ -77,6 +77,12 @@ bash build.sh
 cd ../../..
 ```
 
+If the build fails, maybe you need to install

```

Comment:
@lytico in which way does the build fail when this pkg is missing? Any error message?

Reply:
i didn't record the exact error message, it was just something like
`build failed with error ....`
it had no clear relation to the error source, instead i had to poke it out from a build log file.